### PR TITLE
Update reports and team-reports zero state

### PR DESCRIPTION
**Pull request by yksolanki9**
Reports
Before:
localhost_8100_enterprise_my_reports_filters=%7B%7D(iPhone 6_7_8 Plus) (3)

After:
localhost_8100_enterprise_my_reports_filters=%7B%7D(iPhone 6_7_8 Plus) (6)

Team reports:
Before:
localhost_8100_enterprise_my_reports_filters=%7B%7D(iPhone 6_7_8 Plus) (2)

After:
localhost_8100_enterprise_my_expenses_filters=%7B%7D(Moto G4) (5)

### CODE REVIEW

File: src/app/fyle/team-reports/team-reports.page.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <ion-buttons mode="md" slot="start">
       <app-fy-menu-icon></app-fy-menu-icon>
     </ion-buttons>
-    <ion-title mode="md" class="page-title"> Reports </ion-title>
+    <ion-title mode="md" class="page-title"> Reports - Team </ion-title>
```
Comment (Dimple16) [inline src/app/fyle/team-reports/team-reports.page.html:6]: should be `Team Reports`

File: src/app/fyle/team-reports/team-reports.page.html [hunk 2]
```diff
@@ -93,9 +93,11 @@
   <ng-template #zeroState>
     <div class="reports--zero-state">
       <app-fy-zero-state
-        [image]="'../../../assets/images/zero-states/coffee.svg'"
-        [header]="'Nothing here, grab a coffee maybe?'"
-        [message]="'Expense reports that need your approval will show up here.'"
+        [unscaledImage]="true"
+        [useNewStyling]="true"
+        image="../../../assets/images/zero-states/expenses.png"
+        header="No Reports found !"
```
Comment (Dimple16) [inline src/app/fyle/team-reports/team-reports.page.html:99]: I guess the content here is not updated based on new design, can you verify this - https://www.figma.com/file/7QTRyXwGZWWG8BCwcTAzob/%5BMobile%5D-Reports?node-id=705%3A125180

**Review by Dimple16 — CHANGES_REQUESTED** [review]
few comments

**Comment by yksolanki9** [comment]
Merged with other pr

**Pull request closed without merging**